android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -42,19 +45,17 @@ public final class ExoPlayerView extends FrameLayout implements AdViewProvider {
     private final ViewGroup.LayoutParams layoutParams;
     private final FrameLayout adOverlayFrameLayout;
 
-    private boolean useTextureView = true;
-    private boolean useSecureView = false;
     private boolean hideShutterView = false;
 
-    public ExoPlayerView(Context context) {
-        this(context, null);
+    public ExoPlayerView(Context context, @ViewType.ViewType int viewType) {
+        this(context, null, viewType);
     }
 
-    public ExoPlayerView(Context context, AttributeSet attrs) {
-        this(context, attrs, 0);
+    public ExoPlayerView(Context context, AttributeSet attrs, @ViewType.ViewType int viewType) {
+        this(context, attrs, 0, viewType);
     }
 
-    public ExoPlayerView(Context context, AttributeSet attrs, int defStyleAttr) {
+    public ExoPlayerView(Context context, AttributeSet attrs, int defStyleAttr, @ViewType.ViewType int viewType) {
```

Comment:
Do we really need viewType every single constructor?
I guess being able to modify via `updateSurfaceView` means that we don't necessarily need to inject initial values externally.

Reply:
Exact, just push a fix for that !